daemon/src/to_sse_event.rs

```diff
@@ -103,7 +103,10 @@ impl ToSseEvent for CfdsWithCurrentPrice {
             .cfds
             .iter()
             .map(|cfd| {
-                let (profit_btc, profit_usd) = cfd.profit(current_price).unwrap();
+                let (profit_btc, profit_in_percent) = cfd
+                    .profit(current_price)
+                    .map(|(p_l, in_percent)| (p_l.round_dp(6), in_percent.round_dp(2)))
+                    .unwrap_or((Decimal::ZERO, Decimal::ZERO));
```

Comment:
Masking failures here with a default value will be a debugging nightmare. Can we at least add a log statement here?